tests/Tapper.Test.SourceTypes/CollectionClasses.cs

```diff
@@ -5,7 +5,7 @@
 using System;
 using System.Collections.Generic;
 
-namespace Tapper.Tests.SourceTypes;
+namespace Tapper.Test.SourceTypes;
```

Comment:
Don't change namespace. The project name should be changed to Tests, but this should be done outside this PR.

Reply:
The issue here was that all other files use `Tapper.Test.SourceTypes` instead, and the tests only imported `Tapper.Tests.SourceTypes`.
I agree that it should not be part of the PR, but the namespaces were inconsistent and caused some issues when trying to use the enum type, which is why I decided to align them to be consistent at least.

Reply:
I don't think there is a problem, since there hasn't been a problem before. What exactly is the problem?

Reply:
it's not a really a problem, the namespaces are just inconsistent.
I could also import the namespace `Tapper.Test.SourceTypes` alogn with the previously already imported namespace `Tapper.Tests.SourceTypes`, but wouldn't that be confusing?

If you don't agree to align the namespace then I'll revert the change here.

Reply:
In this PR, please revert.